units/host_allocator.go

```diff
@@ -198,6 +199,25 @@ func (j *hostAllocatorJob) Run(ctx context.Context) {
 	// host-spawning phase
 	//////////////////////
 
+	numIntentHosts, err := host.CountIntentHosts(ctx)
+	if err != nil {
+		grip.Error(message.WrapError(err, message.Fields{
+			"runner":   hostAllocatorJobName,
+			"instance": j.ID(),
+			"distro":   j.DistroID,
+			"message":  "failed to count intent hosts",
+		}))
+	}
+	if numIntentHosts > maxIntentHosts {
+		grip.Info(message.Fields{
+			"runner":   hostAllocatorJobName,
+			"instance": j.ID(),
+			"distro":   j.DistroID,
+			"message":  "too many intent hosts, skipping host allocation",
```

Comment:
Should we add the maxIntentHosts and the numIntentHosts to this grip? I think it could help debug when it's hit, what Evergreen thought the numbers were at the time